src/flir_adk_ethernet/EthernetCamera.cpp

```diff
@@ -92,6 +105,11 @@ bool EthernetCamera::findMatchingCamera(CameraListWrapper camList, const unsigne
         CameraWrapper cam = camList.GetByIndex(i);
         INodeMap &nodeMapTLDevice = cam.GetTLDeviceNodeMap();
 
+        CBooleanPtr wrongSubnet = nodeMapTLDevice.GetNode("GevDeviceIsWrongSubnet");
+        if (wrongSubnet->GetValue()) {
+            continue;
```

Comment:
maybe add debug or warning about this? in cases where no matching subnets are found, it might be confusing that nothing was selected, etc.